lib/index.js

```diff
@@ -295,9 +295,9 @@ Table.prototype.toString = function (){
   * @return String splitted on several lines.
   */
   function beautify_string(content, wrap_line_length) {
-    // - 2 for padding, borders, etc. 
-    var wrap_value = (wrap_line_length - 2) || 250;
-    var symbolsToAdd =  extendWrapLine(content, wrap_line_length);
+    var padding = (style['padding-left'] || 0) + (style['padding-right'] || 0);
+    var wrap_value = wrap_line_length && wrap_line_length > padding ? wrap_line_length - padding : 250;
```

Comment:
This should be `var wrap_value =  wrap_line_length > padding ? wrap_line_length - padding : 250`